catalyst_voices/packages/catalyst_voices_blocs/lib/src/brand/brand_event.dart

```diff
@@ -0,0 +1,17 @@
+part of 'brand_bloc.dart';
+
+abstract final class BrandEvent extends Equatable {
+  const BrandEvent();
+
+  @override
+  List<Object> get props => [];
+}
+
+final class BrandChanged extends BrandEvent {
```

Comment:
Commonly the pattern for Bloc events was to add 'Event' Suffix to them so they could be easily identified, like 'BrandChangedEvent'. Should we follow this pattern here too?